docs/source/incompressible_viscous_basics.rst

```diff
@@ -0,0 +1,80 @@
+Incompressible hydrodynamics solver
+===================================
+
+pyro's incompressible solver solves:
+
+.. math::
+
+   \frac{\partial U}{\partial t} + U \cdot \nabla U + \nabla p &= 0 \\
+   \nabla \cdot U &= 0
+
+The algorithm combines the Godunov/advection features
+used in the advection and compressible solver together with multigrid
+to enforce the divergence constraint on the velocities.
+
+Here we implement a cell-centered approximate projection method for
+solving the incompressible equations. At the moment, only periodic BCs
+are supported.
+
+The main parameters that affect this solver are:
+
+.. include:: incompressible_viscous_defaults.inc
+
+Examples
+--------
+
+cavity
+
+The lid-driven cavity is a well-known benchmark problem for hydro codes
+(see e.g. )
+^^^^^
+
+The shear problem initializes a shear layer in a domain with
+doubly-periodic boundaries and looks at the development of two
+vortices as the shear layer rolls up. This problem was explored in a
+number of papers, for example, Bell, Colella, & Glaz (1989) and Martin
+& Colella (2000). This is run as:
```

Comment:
You should add the full citations to `docs/source/refs.bib` and reference them with `` :cite:`...` ``.

Reply:
This is just copy-pasted from incompressible docs, I was going to change it later

Reply:
Ah, I'll fix it there then